scripts/dev/install_prerequisites.sh

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
 
 if ! command -v telepresence &> /dev/null; then \
-	echo "Telepresence not found, installing now"
-	sudo curl -fL https://app.getambassador.io/download/tel2/darwin/amd64/latest/telepresence -o /usr/local/bin/telepresence
-	sudo chmod a+x /usr/local/bin/telepresence
+  echo "Telepresence not found, installing now"
+  if [[ $(uname -m) == 'arm64' ]]; then
+      echo "arm64"
+      sudo curl -fL https://app.getambassador.io/download/tel2/darwin/arm64/latest/telepresence -o /usr/local/bin/telepresence
+  else
+      echo "amd64"
+      sudo curl -fL https://app.getambassador.io/download/tel2/darwin/amd64/latest/telepresence -o /usr/local/bin/telepresence
```

Comment:
If I'm not mistaken, this could be shortened to `curl https://app.getambassador.io/download/tel2/$(go env GOHOSTOS)/$(go env GOHOSTARCH)/latest/telepresence`.

Also, please first download the binary and them move it to `/usr/local/bin`. Otherwise you're risking writing invalid data straight to your `/usr/local/bin`.

Reply:
Did you have something like this on your mind?

```
  sudo curl -fL https://app.getambassador.io/download/tel2/"$(go env GOHOSTOS)"/"$(go env GOHOSTARCH)"/latest/telepresence -o /tmp/telepresence
  sudo mv /tmp/telepresence /usr/local/bin/telepresence
  sudo chmod a+x /usr/local/bin/telepresence
```

Reply:
For now I added the patch here: https://github.com/mongodb/mongodb-kubernetes-operator/pull/1184/commits/e07dbd22e55999d2d06bec1d7b2c678e6c5b24f0 let me know if meant it different!